fe/fe-core/src/test/java/com/starrocks/analysis/CreateMaterializedViewTest.java

```diff
@@ -3960,4 +3960,21 @@ public void testCreateMvWithUnsupportedStr2date() {
                     AnalysisException.class, () -> starRocksAssert.useDatabase("test").withMaterializedView(sql));
         }
     }
+
+    @Test
+    public void testCreateMvWithDataCachePartitionDuration() {
+        {
+            String sql = "create materialized view mv1 " +
+                    "partition by ss " +
+                    "distributed by hash(a) buckets 10 " +
+                    "REFRESH DEFERRED MANUAL " +
+                    "PROPERTIES (\n" +
+                    "\"replication_num\" = \"1\"\n" +
+                    "\"datacache.partition_duration\" = \"6 day\"" +
+                    ") " +
+                    "as select str2date(d, '%m-%d-%Y') ss, a, b, c from jdbc0.partitioned_db0.tbl1;";
+            Assert.assertThrows("Materialized view partition function date_trunc check failed",
```

Comment:
why just add an invalid case, should cover the success case, check if the partition_duration is set correctly.

UT is not just for code coverage, should be always function oriented.

Reply:
OK